setup.md

```diff
@@ -85,11 +102,13 @@ That's OK! There’s a dialog box hiding behind RStudio asking if you want to cr
 
 ![](figure/general_pkg_output.png#inline-img "console output")
 
-# Grab the sample datasets
+# Grab the Sample Datasets
+
+We'll be working with a chemistry dataset and metadata from the Bight for many of our examples. Please download the chemistry dataset [here](https://github.com/SCCWRP/SCCWRP_R_training/blob/master/data/B13%20Chem%20data.xlsx?raw=true) and the metadata [here](https://github.com/SCCWRP/SCCWRP_R_training/blob/master/data/Master%20Data%20-%20Station%20Info.xlsx?raw=true).  Make sure you have access to these data on the day of training.
 
-We'll be working with a chemistry dataset and metadata from the Bight for many of our examples. Please download the chemistry dataset [here](http://bit.ly/2Gpgzil) and the metadata [here](http://bit.ly/2sCBtbV).  Make sure you have access to these data on the day of training.
+<!-- may want to make .csv versions of these data so folks can see what it looks like and download more easily? Or is this and older version, we'll be using the calipsa/ascidat data in the CABW2018_R_training repo? -->
```

Comment:
Yea the text on line 107 is from some old material, it will need to be updated.  I'll take care of this on the setup branch you created.